payments-core/src/main/java/com/stripe/android/view/CardWidgetViewModel.kt

```diff
@@ -29,13 +29,22 @@ import javax.inject.Provider
 internal class CardWidgetViewModel(
     private val paymentConfigProvider: Provider<PaymentConfiguration>,
     private val stripeRepository: StripeRepository,
-    dispatcher: CoroutineDispatcher = Dispatchers.IO,
+    private val dispatcher: CoroutineDispatcher = Dispatchers.IO,
 ) : ViewModel() {
 
     private val _isCbcEligible = MutableStateFlow(false)
     val isCbcEligible: StateFlow<Boolean> = _isCbcEligible
+    var onBehalfOf: String? = null
```

Comment:
We should set this in savedStateHandle too.

Reply:
Done 👍